src/HermesSerializers/index.js

```diff
@@ -40,6 +43,21 @@ class HermesSerializers {
     unserializeArgs(args) {
         return this._serializers.reduce((args, serializer) => serializer.unserialize(args), args);
     }
+
+    checkPosibiltySend(object, path = "arguments") {
+        if (typeof object === "function") throw new Error(`You try to send a function in worker, this is not possible, please remove function at ${path}`);
```

Comment:
Remove "in worker"

Reply:
What about "Cannot send a function in worker, please remove it a ${path}" ?

Reply:
Yes it's good but, this error is throw also if worker try send function to page.

I suggest "Cannot send a function in other thread, please remove it a ${path}"

When do you say?

Reply:
"Worker cannot sent or receive any function, please remove it a ${path}" then ? 

You could also change the log for Error accordingly

Reply:
Yes ! 👍

Reply:
"Worker cannot send*"

Reply:
"remove it at*" ?

Reply:
@Krenodeno it's ok for u ?

Reply:
saw another typo, but for this one, it's all good ;)